### PR TITLE
slingshot: add umbrella README

### PR DESCRIPTION
**Pull request by oleganza**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Project Slingshot
+
+_Accelerating trajectory into the interstellar space._
```
Comment (bobg) [inline README.md]: s/the //

File: README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Slingshot is a new blockchain architecture under heavy development,
```
Comment (bobg) [inline README.md]: s/heavy/active/

File: README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+with strong focus on scalability, privacy and safety.
```
Comment (bobg) [inline README.md]: "with a strong focus"

File: README.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+The Slingshot project consists of the following sub-protocols and components:
+
+### [Slidechain](slidechain)
+
+This is Slidechain, a demonstration of a minimal [Stellar](https://stellar.org/) sidechain.
```
Comment (bobg) [inline README.md]: s/This is Slidechain, a/A/

File: README.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+Slidechain allows you to _peg_ funds from the Stellar testnet, _import_ then to a _sidechain_,
+and later _export_ them back to Stellar.
+
+* [Slidechain README](slidechain/Readme.md)
+* [Pegging specification](slidechain/Pegging.md)
+* [Demo documentation](slidechain/Running.md)
+
+### [ZkVM](zkvm)
+
+An evolution of [TxVM](https://github.com/chain/txvm) with **cloaked assets** and **zero-knowledge smart contracts**.
```
Comment (bobg) [inline README.md]: s/An/A blockchain transaction format that is an/

File: README.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+* [README](zkvm/README.md)
+* [Specification](zkvm/docs/zkvm-spec.md)
+* [API guide](zkvm/docs/zkvm-api.md)
+
+### [Spacesuit](spacesuit)
+
+Interstellar’s implementation of _Cloak_, a confidential assets protocol
+based on the Bulletproofs zero-knowledge circuit proof system.
```
Comment (bobg) [inline README.md]: Include a link for "Bulletproofs"?

Comment (oleganza) [reply README.md]: yup!

**Pull request merged**